### PR TITLE
Support formatting labels in charts in Basic UI

### PR DESCRIPTION
**Pull request by J-N-K**
Depends on https://github.com/openhab/openhab-core/pull/2937
Related to https://github.com/openhab/openhab-core/issues/1033

Signed-off-by: Jan N. Klug <github@klug.nrw>

### CODE REVIEW

**Comment by lolodomo** [comment]
I will merge that PR once related PR in core is merged

**Comment by J-N-K** [comment]
I guess we need a new core-snapshot before GitHub actions pick up the new artifacts.

**Comment by kaikreuzer** [comment]
Already built it.

**Review by kaikreuzer — APPROVED** [review]

**Pull request merged**
